src/main/resources/assets/computercraft/lua/rom/programs/lua.lua

```diff
@@ -19,15 +19,25 @@ local tEnv = {
 setmetatable( tEnv, { __index = _ENV } )
 
 local function displayTable( tDisplay )
```

Comment:
The function is called displayTable but returns a string instead of doing its name sake.

If the only use of the resulting string is to print it (for this program at least) then in my mind the function should do the actual printing. If the output string does get used for something else then this function should be renamed.